lib/bearerstrategy.js

```diff
@@ -293,8 +293,9 @@ Strategy.prototype.authenticate = function authenticateStrategy(req) {
 
       var metadataURL = self._options.identityMetadata;
 
-      if (self._options.policyName) {
-        log.info('B2C: using policy %s', self._options.policyName);
+      if (!self._options.policyName) {
```

Comment:
You are right, but we have a bigger problem. I had two code paths - one for B2C and one for v2. It seems that this has been removed. A lot is now broken if that is the case, because this code now:

* Automatically does changes to the metadata endpoint (removes common)
* Automatically uses policyName when evaluating policies.

We need to have two forks in the waterfalls like I had in the old code base. I'm open to other options as well.

Right now this is effectively a B2C BearerStrategy. 🎱